packages/gasket-plugin-nextjs/generator/next/server.mjs

```diff
@@ -0,0 +1,7 @@
+import gasket from './gasket.mjs';
+
+async function main() {
+  await gasket.actions.startServer();
+}
```

Comment:
Is awaiting necessary here?
```suggestion
function main() {
  gasket.actions.startServer();
}
```

Reply:
That function is async... https://github.com/godaddy/gasket/pull/725/files#diff-1ad8f0a34128ce187e4dff25274f52b81634ad2ea45db507006ebab859f21dceR76-R83

We'd need to update every lifecycle to change that. There's I think 8 or so in that file.

Reply:
No I just mean in this server.mjs file. No real reason for the await. For example if this were TS we might write it as 
```
void gasket.actions.startServer();
```

Reply:
Ahh yes I get what you're saying. Also should these be `.js`?